src/Scriban/Syntax/Expressions/ScriptBinaryOperator.cs

```diff
@@ -191,6 +194,10 @@ public static string ToText(this ScriptBinaryOperator op)
                     return "&";
                 case ScriptBinaryOperator.BinaryOr:
                     return "|";
+                case ScriptBinaryOperator.IsEmpty:
```

Comment:
I think that not correct. But I'm not sure where should I include the function. LiquidBuiltinsFunctions class, TryLiquidToScriban method?
I doubt that there are anything like whitespace of empty in liquid language. So that the question.